usaspending_api/etl/es_transaction_template.json

```diff
@@ -255,7 +260,12 @@
         },
 
         "recipient_location_country_code": {
-          "type": "text"
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword"
+            }
+          }
```

Comment:
Why does this have a keyword field on the inside, instead of just having type keyword?

Reply:
Elsewhere in the code `{scope}_country_code.keyword` is called, so adding a keyword type won't require changing the code

Reply:
Do we know if there are any performance issues with having two fields (I'm assuming the separate type makes ES store/index the variable as both text and keyword) associated with country code? Changing the references might be a better choice if we're hurting performance just to minimize lines of changes.
@kbard since you would likely know more about this than me.

Reply:
I've checked it both ways on my local elasticsearch cluster and there was basically no difference (~10 ms in favor of the seperate keyword)